vendor/binary-install/index.js

```diff
@@ -1,106 +0,0 @@
-// we directly include this in our project so that the downloaded binary is stored
-// with our module instead of with some other one. This allows us to avoid using
-// nodejs as a wrapper executable to run our binary, greatly speeding up performance
-// see: https://github.com/EverlastingBugstopper/binary-install/tree/main/packages/binary-install
-
-const { existsSync, mkdirSync } = require("fs")
```

Comment:
Why did you delete this file?

I'm pretty sure this file must not be deleted or it will break the install